omnibus-arbitrum-mainnet.toml

```diff
@@ -297,7 +297,7 @@ args = [
     "<%= settings.account_withdraw_timeout %>",
 ]
 
-[setting.spartan_council_pool_id_setting]
+[setting.sc_pool_id]
```

Comment:
yeah this will cause issues if we change it? It's not referenced anywhere with `spartan_council_pool_id_setting` ?

Reply:
also why setting 😅 ?

Reply:
It's used in collateral configuration, with `sc_pool_id` only

Reply:
this is wrong toml. Not on base, but arb. 
On arb this setting is called spartan_council_pool_id_setting and should not be changed, the build fails because pool creation uses `settings.spartan_council_pool_id_setting`

Reply:
Hoping it's resolved now https://github.com/Synthetixio/synthetix-deployments/pull/506/commits/c4e33fd0f114eee34a4e5f58ae7c254b9be4aa7f

Reply:
it is not. This change we are talking about here is for arbitrum mainnet toml. Needs to be reverted.

Reply:
finally resolved 😅